app/packages/looker/src/worker/painter.ts

```diff
@@ -226,11 +228,19 @@ export const PainterFactory = (requestColor) => ({
       const cache = {};
 
       let color;
-      if (maskTargets && Object.keys(maskTargets).length === 1) {
-        color = get32BitColor(
-          setting?.fieldColor ??
-            (await requestColor(coloring.pool, coloring.seed, field))
-        );
+      let colorKey;
+      if (maskTargets && Object.keys(maskTargets).length === 0) {
+        if (coloring.by === "field") {
```

Comment:
yet to test this code, but is `Object.keys(maskTargets).length === 0` over `Object.keys(maskTargets).length === 1` intentional here?

Reply:
Great catch. It was intentional, but I saw the bug after testing it. 
Just fixed it. thank you!